### DISCUSSION TITLE
[Q/A][Java 1.8 )] alibaba cloud dubbo provider and not-alibaba cloud dubbo consumer problem

### DISCUSSION THREAD

**Discussion by jk9991xx** [apache/dubbo#14765] [Question]
- [X] I am sure that all the content I provide is in English.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Details

dubbo provider is alibaba cloud dubbo and it implements with nacos.
two consumers:

              1.alibaba cloud dubbo consumer registeration info as below, it works,

Register: consumer://192.168.1.73/org.apache.dubbo.rpc.service.GenericService?application=base-auth&category=consumers&check=false&dubbo=2.0.2&generic=true&group=dx-front&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&pid=22500&qos.enable=false&release=2.7.8&router=-default,revisionRouter&side=consumer&sticky=false&timestamp=1728643191548&version=1.0.0, dubbo version: 2.7.8, current host: 192.168.1.73

         2. non-alibaba cloud dubbo consumer register info as below, it doesn't work,
Register: consumer://192.168.1.9/com.prd.ast.dx.front.dubbo.module.user.service.IUserService?application=first-dubbo-client&category=consumers&check=false&dubbo=2.0.2&interface=com.prd.ast.dx.front.dubbo.module.user.service.IUserService&metadata-type=remote&methods=list&pid=186548&release=2.7.8&revision=1.0&side=consumer&sticky=false&timestamp=1728644774241, dubbo version: 2.7.8, current host: 192.168.1.9

  provider env ,consumer 1 env: there are same
	 java:1.8
	 <spring-boot.version>2.3.2.RELEASE</spring-boot.version>
         <spring-cloud.version>Hoxton.SR10</spring-cloud.version>
         <spring-cloud-alibaba.version>2.2.6.RELEASE</spring-cloud-alibaba.version>
     	 <dubbo.version>2.7.8</dubbo.version>
	
 consumer 2 env:
	 java: 1.8
         <dependency>
             <groupId>org.apache.dubbo</groupId>
             <artifactId>dubbo-registry-nacos</artifactId>
             <version>2.7.8</version>
         </dependency>
         <dependency>
             <groupId>com.alibaba.spring</groupId>
             <artifactId>spring-context-support</artifactId>
             <version>1.0.10</version>
         </dependency>
        <dependency>
             <groupId>org.apache.dubbo</groupId>
             <artifactId>dubbo-spring-boot-starter</artifactId>
             <version>2.7.8</version>
         </dependency>
        
 two registeation url are different,how can I make consumer 2 work? thanks. 

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment]
What is the problem? Consumer info will not effect the usage
